### DISCUSSION TITLE
Summarize your question into a single sentence here

### DISCUSSION THREAD

**Discussion by Cryptics1999** [GriefPrevention/GriefPrevention#964] [Q&A] [answered]
Max Accrued Claim Blocks:
      Default: 8000
      VIP: 10000
      VIP+: 12000
      Prince: 14000
      King: 18000
      Emperor: 22000
      Deity: 25000

Is this how you change the acruel and what is the permission node for these if so

**Comment by Cryptics1999** [comment]
Claim Blocks Accrued Per Hour:
      Default: 100
      VIP: 200
      VIP+: 250
      Prince: 300
      King: 400
      Emperor: 420
      Deity: 440

**Comment by Jikoo** [comment, marked as answer]
You need to use [GPAccruals](https://github.com/TechFortress/GriefPrevention/wiki/Addons#gpaccruals) for multiple rates/max caps.
